recipes/omicverse/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "omicverse" %}
+{% set version = "1.4.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1cab32bc2e87dc3d70785335480d2e70834cf66461daf504f33eadb265833cb0
+
+build:
+  skip: true  # [win or py<38]
```

Comment:
```suggestion
  python: noarch
```

Reply:
Hi, @mwcraig 

Thank you for your suggestion. I have updated this project to be a `noarch` project.

Zehua